### ISSUE TITLE
Create script to generate oracle target data for evaluation

### ISSUE THREAD

**Issue by bsweger** [reichlab/variant-nowcast-hub#215]
**Background**

When we're ready to score the models for a variant-nowcast-hub round, we need produce target data in [oracle output format](https://hubverse.io/en/latest/user-guide/target-data.html#oracle-output).

The "oracle output" file for each round will be derived from the time series version of the target data. In other words, the time series file needs to be created first, so this issue is dependent on #212 

**Definition of done**

- [ ] Script is in repo's `src` directory
- [ ] Script contains a function that generates an oracle output version of the target data, which will have the following columns: nowcast_date (i.e., round_id), location, target_date, clade, oracle_value. The oracle_value is the count of the clade for the given location and target_date.
- [ ] Script parameter is the location of target time series data that will be used to derive the oracle output and/or the nowcast_date/round_id
- [ ] Oracle target data is saved to `target-data/oracle-output/nowcast_date={nowcast_date}/part-n.parquet`

Out of scope for this issue is automating the script to run each Wednesday (see #216)
